boards/px4/fmu-v3/init/rc.board_defaults

```diff
@@ -4,7 +4,28 @@
 #------------------------------------------------------------------------------
 
 
+#
+# Bootloader upgrade
+#
```

Comment:
Unintentional addition?

Reply:
No, trying to keep fmu-v2 and fmu-v3 mostly in sync. It's still conceivable you might want or need to update the bootloader on fmu-v3.

Reply:
Ok, then please split the commit, otherwise anyone reading history later would have no clue.